RetrospectiveExtension.Frontend/config/environment.tsx.template

```diff
@@ -0,0 +1,7 @@
+const environment = {
```

Comment:
I appreciate this being template-able! 👏

Reply:
done